test/unit/test_runner_config.py

```diff
@@ -606,19 +607,22 @@ def test_containerization_settings(mock_mkdir, container_runtime):
         mock_containerized.return_value = True
         rc.prepare()
 
-    extra_container_args = []
+    extra_global_args = []
+    extra_run_args = []
     if container_runtime == 'podman':
-        extra_container_args = ['--quiet']
+        extra_global_args = [f'--root={str(Path.home())}/.local/share/containers/storage']
+        extra_run_args = ['--quiet']
     else:
-        extra_container_args = ['--user={os.getuid()}']
+        extra_run_args = ['--user={os.getuid()}']
```

Comment:
is this use of `--user=<some user id>` and option that we should add to docs? this is to avoid the files getting owned by root if using docker, right?